nov-13-hardfork-spec.md

```diff
@@ -0,0 +1,38 @@
+# November 13th Bitcoin Cash Hardfork Technical Details
+
+Version 1.0, 2017-12-01
+
+## Summary
+ 
+When the median time past[1] of the most recent 11 blocks (MTP-11) is greater than or equal to UNIX timestamp 1510600000 Bitcoin Cash will execute a hardfork according to this specification.  It includes the following three protocol changes:
```

Comment:
> will execute a hardfork according to this specification

Let's elaborate on that for precision's sake.

"will execute a hardfork according to this specification. Starting from the next block these three consensus rules changes will take effect:"

or something of that sort which leaves no doubt when the new rules take effect.